README.md

```diff
@@ -176,17 +177,24 @@ Variable `RMW_CONNEXT_DISABLE_FAST_ENDPOINT_DISCOVERY` may be used to disable
 these automatic optimizations, and to leave the DomainParticipant's QoS to
 its defaults.
 
-### RMW_CONNEXT_USE_DEFAULT_PUBLISH_MODE
+### RMW_CONNEXT_ENDPOINT_QOS_OVERRIDE_POLICY
 
-`rmw_connextdds` will always set `DDS_DataWriterQos::publish_mode::kind` of
-any DataWriter it creates to `DDS_ASYNCHRONOUS_PUBLISH_MODE_QOS`, in order to
-enable out of the box support for "large data".
+When this variable is not set or set to `always`, the qos settings specified in the default profile will be used and the ros qos profile will be applied on top of it.
+You can use topic filters in XML profile files to have different defaults for different topics, but you have to use the mangled topic names (see [ROS topic mangling conventions](#ros-topic-mangling-conventions)).
 
-This behavior might not be always desirable, and it can be disabled by setting
-`RMW_CONNEXT_USE_DEFAULT_PUBLISH_MODE` to a non-empty value.
+In case this variable is set to `never`, the qos settings will be loaded from the default profile as before but the ros qos profile will be ignored.
+Be aware of configuring the qos of rcl topics (rt/rosout, rt/parameter_events, etc) and the rmw internal topic `ros_discovery_info` correctly.
 
-This variable is not used by `rmw_connextddsmicro`, since it doesn't
-automatically override `DDS_DataWriterQos::publish_mode::kind`.
+This variable can also be set to `dds_topics: <regex>`, e.g.: `dds_topics: rt/my_topic|rt/my_ns/another_topic`.
+In that case, qos settings for topics matching the provided regex will be loaded in the same way as the `never` policy, and the ones that don't match will be loaded in the same way as the `always` policy.
+
+#### ROS topic mangling conventions
+
+ROS mangles topic names in the following way:
+
+- Topics are prefixed with `rt`. e.g.: `/my/fully/qualified/ros/topic` is converted to `rt/my/fully/qualified/ros/topic`.
+- The service request topics are prefixed with `rq` and suffixed with `Request`. e.g.: `/my/fully/qualified/ros/service` request topic is `rq/my/fully/qualified/ros/serviceRequest`.
+- The service response topics are prefixed with `rr` and suffixed with `Response`. e.g.: `/my/fully/qualified/ros/service` response topic is `rr/my/fully/qualified/ros/serviceResponse`.
```

Comment:
I think this section looks good as is, at least for an initial merge.

I'm considering starting a proper manual in Sphinx for `rmw_connextdds[micro]`, putting it on public hosting (e.g. readthedocs), and move most documentation out of the readme, to avoid bloating it inevitably.

With that available, I would probably move this information about topic mangling to a different section, one more general than this one about env variables (e.g. "Configuring QoS in `rmw_connextdds`").